alpine-nix-rails-nginx/etc/nginx/dynamic-content.conf

```diff
@@ -0,0 +1,20 @@
+# Disable caching for dynamic content
+expires off;
+
+# Work-around for BREACH attack: http://breachattack.com/
+gzip off;
+
+# Restrict allowed methods
+if ($request_method !~ ^(DELETE|GET|HEAD|OPTIONS|PATCH|POST|PUT)$) {
+  more_set_headers 'Allow: DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT';
+  return 405;
+}
+
+# Handle OPTIONS requests
+if ($request_method = OPTIONS) {
+  more_set_headers 'Allow: DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT';
+  return 204;
+}
+
+# Rate Limit
+limit_req zone=dynamic burst=5 nodelay;
```

Comment:
Should we set some default rate limiting that is reasonable? I mean, most apps probably want to prevent someone from hammering the dynamic parts of the site. The question is what is a good generic limit?